kubevirt_handler/metadata.csv

```diff
@@ -0,0 +1,47 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
```

Comment:
Could probably be good to add some units here as well

Reply:
Yep, I was just looking at them right now, I will add them to the kubevirt_controller integration as well!

Reply:
Done!